data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2403 @@
+			choice
+				`	(Go closer to see what is going on.)`
+				`	(Leave. This does not seem interesting.)`
+					decline
+			`	As you move closer you realize that the colours of the circles must matter too, but you are not sure in what way. The Unfettered are not all fighting with the same weapons. They move between circles, like in a huge dance. Some of them seem to have standard Hai weapons, but it does not look like they are deadly.`
+			choice
+				`	(Join the fight!)`
+					goto fight
+				`	(Keep looking.)`
+			`	You keep looking around and are eventually attracted towards the bigger fight, the one He Yi is in. He seems to be handling 6 younger Unfettered all by himself, and you most admit he is doing quite a good job at it.`
+			choice
+				`	(Challenge him.)`
+					goto challenge
+				`	(Try to speak to him.)`
+					goto speak
+				`	(Join another less risky fight.)`
+			
+			label fight
+			`	You look at different fights and see one with less experienced, younger Unfettered, they seem to be fighting in a less crowded area than the rest. As you come closer they see your interest and propose you to join the fight.`
+			choice
+				`	(Accept.)`
+				`	(Decline.)`
+					goto nope
+			`	One of them comes towards you. "I challenge you, <last>! Tales of your strength have been said, I want to test it out first-hand. If I win, I demand you tell me those stories yourself." He is equipped with what looks to be a twin pistol version of the standard Hai rifles.`
+			choice
+				`	"I am sure those tales are only half as good as they should be!"`
+				`	"Very well, but if I win, I want your weapon."`
+					goto weapon
+			`	"I see the tales about your wisdom were exaggerated at least." Now it is time for the challenge to begin.`
+			
+			label weapon
+			`	He looks at his weapons, thinking. Then he nods and hands you one of the pistols. "I have set it on training. We would not want any accidents to happen," he adds. Then he moves away to the edge of the blue circle. "Because this is your first time, I will let you choose your position, and have the initiative. The rules are simple: once you shout the fight begins." You understand why you heard so much shouting earlier.`
+			choice
+				`	(SHOUT!)`
+					goto start
+				`	(Smartly get into position and get ready.)`
+			`	You carefully observe your surroundings and decide to head towards one of the smaller circles, not too close yet not too far from him, and consider your covers. Everything is simulated to resemble an urban training, probably because the ancestors of the Unfettered had to protect their cities from the Korath. You feel the adrenaline of the coming battle surging into your veins.`
```

Comment:
```suggestion
			`	You carefully observe your surroundings and decide to head towards one of the smaller circles, not too close yet not too far from him, and consider your covers. Everything is simulated to resemble an urban environment, probably because the ancestors of the Unfettered had to protect their cities from the Korath. You feel the adrenaline of the coming battle surging into your veins.`
```